doc/misc/aliases.7.rst

```diff
@@ -79,9 +79,13 @@ Keys:
     - ``descr`` - Description that will be shown in help.
     - ``group_id`` - A group this alias is part of if any.
     - ``complete`` - Whether bash autocompletion should be used for this alias, default is false.
+    - ``number_of_attached_values`` - Number of arguments consumed by the command alias. These arguments
+      can be used as values for attached named arguments. Default is 0. Added in config file version 1.1.
+      Ignored by versions of DNF that support only config file version 1.0.
     - ``attached_named_args`` - Options that will be used with the command. The format is an array of inline
       tables, each of which must contain an ``id_path`` key to specify the path to an option, and may contain also
-      a ``value`` key to specify the value of the option.
+      a ``value`` key to specify the value of the option. The ``value`` can contain ``${index}`` placeholders,
```

Comment:
Please document that the index counts from 1.

Reply:
OK, I've added it to the documentation.